### DISCUSSION TITLE
metadata: protocols

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#19] [Q&A] [answered]
I am trying to publish dandiset https://dandiarchive.org/dandiset/000008/draft. 

I put https://doi.org/10.1038/nprot.2017.120 as the protocol and I am getting the validation error:

protocol -> 0
Invalid or missing URL scheme (type=valid.url.scheme)

Am I entering this information incorrectly?

**Comment by waxlamp** [comment, marked as answer]
It seems like maybe you need the URL scheme as part of the entered data. My guess is, something like a http:// or https:// prefix might fix the problem.

**Comment by bendichter** [comment]
I don't know what fixed it but it's working now
